www/package.json

```diff
@@ -111,7 +113,7 @@
   "main": "n/a",
   "private": true,
   "scripts": {
-    "build": "gatsby build",
+    "build": "lingui compile && gatsby build",
```

Comment:
Is this the right place to put this? Can we do `prebuild` or something?

Reply:
You could install `husky` and `lint-staged` to run a `pre-commit` hook:
https://github.com/gatsbyjs/gatsby/blob/389484031fad9da72cd739d8a6a9404347280953/package.json#L68-L86

Or does this only need to be run before a build and not necessarily a commit?

Reply:
`lingui extract` is the thing that generates the .po files -- those need to be in the codebase so that translators know what they need to translate, so it needs to be pre-commit.

Should I install husky on www or is there some way to use the one configured on top level?

Reply:
www is separate so to speak, so yeah, go ahead with installing it there 👍

Reply:
I added husky and lint-staged but ran into two issues:

* I can’t figure out how to tell lint-stage to commit the generated po files.
* lingui extract takes a few seconds to run, and doesn’t have any options to only run for specific files.

I don’t think its good to run it on every commit, especially since most www changes will not be string related. I was thinking that on build we could watch the translation files and fail build if there are any changes?